docs/example/saxml/README.md

```diff
@@ -0,0 +1,94 @@
+# Deploy SaxML Multihost with LWS on GKE
+
+Follow the steps on this [guide](https://cloud.google.com/kubernetes-engine/docs/tutorials/tpu-multihost-saxml#before-you-begin) upto the deploy saxml step
+to create GKE cluster, two TPU nodepools, and configure kubectl with the cluster. 
+
+## Install LeaderWorkerSet
+
+Follow the step-by-step guide on how to install LWS. [View installation guide](https://github.com/kubernetes-sigs/lws/blob/main/docs/setup/install.md)
+
+
+## Deploy ConfigMap with model configuration
+
+Apply the `configmaplws.yaml` manifest:
+
```

Comment:
describe what is included in the configmap and why we deploy it

Reply:
Done